src/extensions/scratch3_boost/index.js

```diff
@@ -101,18 +101,30 @@ const BoostPort = {
 };
 
 /**
- * Enum for indexed colors read by the Boost Vision Sensor
+ * Ids for each color sensed by the Boost Vision Sensor
+ */
+const COLOR_ID_ANY = 'any';
+const COLOR_ID_NONE = 'none';
+const COLOR_ID_RED = 'red';
+const COLOR_ID_BLUE = 'blue';
+const COLOR_ID_GREEN = 'green';
+const COLOR_ID_YELLOW = 'yellow';
+const COLOR_ID_WHITE = 'white';
+const COLOR_ID_BLACK = 'black';
+
+/**
+ * Object to look up a color ID by color index
  * @readonly
  * @enum {number}
  */
-const BoostColor = {
-    NONE: 255,
-    RED: 9,
-    BLUE: 3,
-    GREEN: 5,
-    YELLOW: 7,
-    WHITE: 10,
-    BLACK: 0
+const BoostColorIdByIndex = {
```

Comment:
This could probably stay named `BoostColor`, also for the sake of a simpler name, just a thought!